packages/material-ui/src/Tabs/Tabs.js

```diff
@@ -514,6 +522,8 @@ Tabs.defaultProps = {
   component: 'div',
   indicatorColor: 'secondary',
   ScrollButtonComponent: TabScrollButton,
+  scrollButtonLabel: position =>
```

Comment:
@eps1lon Ok, you can review. I'm wondering if it's not overkilling. The more pragmatic approach I can think of is to set `component` to div and to add aria-hidden on the element. The only problem is that VoiceOver call announces "button" when clicking on the button. What do you prefer?

Reply:
I'm wondering more about aria-hidden not working. Could you elaborate what "not working" means in this context? I thought screen readers ignore these elements.

Reply:
Sure, setting aria-label doesn't change the Wave output as well as the fact the VoiceOver announces "button' when clicking on the scroll butons.